### PR TITLE
[#166583061] sets up apollo client and hooks it to terms list

### PR DESCRIPTION
**Pull request by abetss**

### CODE REVIEW

File: beelzebub-web/src/features/terms-notes/scenes/terms-list/terms-list.connector.hook.tsx [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+	}
+	if (error) {
+		return <div>Error! {error.message}</div>;
+	}
+
+	const {
+		terms: { list }
+	} = data;
+
+	return <TermsListComponent terms={list} />;
```
Comment (abetss) [inline beelzebub-web/src/features/terms-notes/scenes/terms-list/terms-list.connector.hook.tsx]: Didn't know how not to extract these components into <TermsListComponent>.
The problem was the useTerm hook was depending on the terms coming from the 
`const { data, error, loading } = useQuery(TERMS_QUERY);` but the data there comes async and we cannot put hooks in an if block. So I could not wrote for example 
```if( !loading ) { ... useHook(data.terms.lis)```

I like to do this without needing to move that those components into a separate component.

Comment (taras101) [reply beelzebub-web/src/features/terms-notes/scenes/terms-list/terms-list.connector.hook.tsx]: I think the issue is that the data should be coming from the cache and not the query.I will look into a refactor

Comment (abetss) [reply beelzebub-web/src/features/terms-notes/scenes/terms-list/terms-list.connector.hook.tsx]: Not sure if that could make a difference, cuz we can access the data from the cache the same way we access the remote data.. but I could be wrong for sure. 
So should I merge this, so you can work on refactoring it?

Comment (taras101) [reply beelzebub-web/src/features/terms-notes/scenes/terms-list/terms-list.connector.hook.tsx]: yes please

**Pull request merged**